convert_results.py

```diff
@@ -0,0 +1,182 @@
+"""This script converts JSON result files to Markdown summaries.
+
```

Comment:
Add a requirements.txt in the directory where the tool is to ensure that we can do pip install -r requirements.txt prior to running the tests.

Reply:
Done